hbase-rsgroup/src/main/java/org/apache/hadoop/hbase/rsgroup/RSGroupAdminServer.java

```diff
@@ -631,6 +631,10 @@ public void updateRSGroupConfig(String groupName, Map<String, String> configurat
     }
   }
 
+  @Override
+  public void updateConfiguration(String groupName) throws IOException {
+  }
```

Comment:
Can you add comment, why empty implementation here

Reply:
Added a comment.